ASM/c/chests.c

```diff
@@ -17,6 +18,11 @@ uint8_t get_chest_override_size(z64_actor_t *actor) {
 		return ((uint8_t*)actor)[0x01E9]; // Chest type
 	}
 
+    // Big chest for GS in 100 GS mode
```

Comment:
The comment says big chest, yet the PR name/return value seem to be a small chest.